utils/api/configurations.js

```diff
@@ -201,16 +201,22 @@ export const configureDocuments = (documents, requestIdentifier) => {
     totalPrice: document.retail_total_price_currency,
     shippingPrice: document.shipping_cost_currency,
     shipTo: {
-      organizationName: document.ship_to.organization_name,
-      text: document.ship_to.text,
+      organizationName: document.ship_to?.organization_name,
+      text: document.ship_to?.text,
     },
     shipFrom: {
-      organizationName: document.ship_from.organization_name,
-      text: document.ship_from.text,
+      organizationName: document.ship_from?.organization_name,
+      text: document.ship_from?.text,
     },
+    // the following properties only need to exist on POs
```

Comment:
since these do not exist on sows, I added an empty string as a fallback. I wasn't sure if it is better to use `configureDocuments` to include all the properties on both POs and SOWs, or to add a separate function called `configurePOs` that calls `configureDocuments` and then adds the PO specific properties to the return object.

Reply:
i changed it to using undefined as a fallback, so i could do this in the component lib: 

`{adPO && <><b>AD PO:</b> {identifier}<br /></>}`

instead of this

`{(adPO.length > 0) && <><b>AD PO:</b> {identifier}<br /></>}`